components/index.ts

```
@@ -1,9 +1,17 @@
+import CustomModal from './CustomModal';
+import FullWidthImage from './FullImage';
 
 export {
   CustomeDatePicker as DatePicker,
   CustomTextInput as TextInput,
   CustomSnackBar as SnackBar,
-}
+  DropdownComponent as Select,
+  CustomImage as Image,
+  CustomModal as Modal,
+  FullWidthImage as FullWidthImage,
```

Biome

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)